src/goose/brokers/redis/console.clj

```diff
@@ -92,14 +119,21 @@
      :periodic  periodic
      :dead      dead}))
 
-(defn enqueued-page-data [redis-conn queue]
-  (let [queues (enqueued-jobs/list-all-queues redis-conn)
+(defn enqueued-page-data
+  [redis-conn queue page]
+  (let [page (Integer/parseInt (or page d/default-page))
```

Comment:
question: why are page numbers strings?

Reply:
`page` is extracted from request params and by default all param values are strings.
Will add validation fn that also conforms the `page` value to `int`.